src/components/shared/Performance.tsx

```diff
@@ -5,6 +5,7 @@ import { makeStyles, Theme } from '@material-ui/core';
 export type PerformanceProps = {
   value: number;
   size?: string;
+  noPaint?: boolean;
```

Comment:
The purpose of `noPaint` is not immediately clear to me. I think `noColor` would be a better name. A docstring would definitely be helpful as well.

Also, there were discussions about moving the color out of `Performance` and `EuroCurrency` entirely. I think this would be a better option than introducing new color-related properties.

If it is absolutely necessary to have these properties, then `Performance` and `EuroCurrency` should at least be consistent, but I don't like `doPaint` in `EuroCurrency` either.

Reply:
I agree. However, I'd advice against refactoring this immediately as this would take some time that I don't think we have until feature freeze. The issue for this is already up under the improvements section.

Reply:
I'd prefer fixing it now instead of introducing new properties that make things worse.